web/tests/event_status.test.js

```diff
@@ -37,6 +37,7 @@ run_test("check_status", async ({override}) => {
         return {
             session: {
                 status: "created",
+                type: "upgrade_from_billing_page",
             },
```

Comment:
Just want to confirm: are there any circumstances in which `type` does not get returned?

Reply:
I have checked the view for this request, and yes, there isn't any case where a `type` is not returned from the request.

Reply:
Except of course when we pass `stripe_payment_intent_id` in request params instead of `stripe_session_id` in which case it returns a `payment_intent` instead of a `session`.